Cargo.toml

```
@@ -57,7 +57,7 @@ tsify-next = { version = ">=0.5.4, <0.6", features = [
 ], default-features = false }
 uniffi = "=0.28.1"
 uuid = { version = ">=1.3.3, <2.0", features = ["serde", "v4"] }
-validator = { version = "0.18.1", features = ["derive"] }
+validator = { version = ">=0.18.1, <1.20", features = ["derive"] }
```

Wait, is this supposed to be this instead?
```suggestion
validator = { version = ">=0.18.1, <0.20", features = ["derive"] }
```

---

Ooops.